local/e2e/backend_test.go

```diff
@@ -78,15 +78,15 @@ func TestLocalBackendRun(t *testing.T) {
 	})
 
 	t.Run("run with ports", func(t *testing.T) {
-		res := c.RunDockerCmd("run", "-d", "-p", "8080:80", "nginx")
+		res := c.RunDockerCmd("run", "-d", "-p", "8888:80", "nginx")
```

Comment:
could we maybe not assign a fixed exposed port and just get the actual random one by inspect response just belond ?